apps/dashboard/test/integration/dashboard_controller_test.rb

```diff
@@ -132,54 +132,56 @@ def teardown
     assert_select "nav a[href='#{batch_connect_sessions_path}']", 0
   end
 
-  test "should not create app menus if NavConfig.categories is empty and whitelist is enabled" do
+  test "should not create app menus if UserConfiguration.categories is empty and filter_nav_categories? is true" do
     SysRouter.stubs(:base_path).returns(Rails.root.join("test/fixtures/sys_with_gateway_apps"))
     OodAppkit.stubs(:clusters).returns(OodCore::Clusters.load_file("test/fixtures/config/clusters.d"))
-    NavConfig.stubs(:categories_whitelist?).returns(true)
-    NavConfig.stubs(:categories).returns([])
+    stub_user_configuration({
+      filter_nav_categories?: true,
+      categories: []
+    })
     
     get root_path
     assert_response :success
     assert_select ".navbar-collapse > .nav li.dropdown[title]", 0
   end
 
-  test "should exclude gateway apps if NavConfig.categories is set to default and whitelist is enabled" do
+  test "should exclude gateway apps if UserConfiguration.categories is set to default and filter_nav_categories? is true" do
```

Comment:
Yea related to the comment above - since `filter_nav_categories` isn't a user supplied variable, what real life production scenario does this test?

Looks like a carry over from previous behavior that we're actually changing now.